test/interface/test_sorting.py

```diff
@@ -109,43 +105,67 @@ def test_invalid_direction(self, session):
 
 class TestSortApplied(object):
 
-    @pytest.mark.usefixtures('multiple_bars_inserted')
+    """Tests that results are sorted only according to the provided
+    filters.
+
+    Excludes nonexplicit sorting from tests: do NOT test how different
+    RDBMS sort rows with the same values being ordered since this is not
+    consistent across RDBMS.
+
+    Excludes `NULL` sorting from tests: sorting fields containing `NULL`
+    values is NOT tested since different RDBMS behave differently when
+    sorting `NULL` values. SQL defines that `NULL` values should be
+    placed together when sorting, but it does not specify whether they
+    should be placed first or last in the result.
+    """
+
+    @pytest.mark.usefixtures('multiple_bars_with_no_nulls_inserted')
     def test_single_sort_field_asc(self, session):
         query = session.query(Bar)
         order_by = [{'field': 'name', 'direction': 'asc'}]
 
         sorted_query = apply_sort(query, order_by)
         result = sorted_query.all()
 
+        result_same_name_1 = [
+            result[0].id, result[1].id, result[2].id, result[3].id
+        ]
+        result_same_name_4 = [result[5].id, result[6].id]
```

Comment:
What's the significance of these variable names?

Reply:
These variables contain a list of results that have the same order. This is to avoid duplicating the list of results multiple times in the test.

I have now renamed these variables to `results_with_same_order_1` and `results_with_same_order_2` since we have two groups of results with the same order in this test. This way, we also avoid including the field value (`name_1`, `name_4`) in the variable name, as it was done, even though `_1` and `_2` are not ideal.

Please let me know if you have any other suggestions.